docs/config.md

```diff
@@ -81,7 +81,10 @@ profile_models: <comma-delimited-string-list>
 [ max_retries: <int> | default: 100 ]
 
 # Specifies how long (seconds) to gather server-only metrics
-[ duration_seconds: <int> | default: 5 ]
+[ duration_seconds: <int> | default: 2 ]
+
+# Specify whether DCGM should be used by Model Analyzer to collect GPU metricss
+[ use_local_gpu_monitor: <bool> | default: False ]
```

Comment:
Can we control the monitoring interval for remote monitor? If no, we can perhaps not expose this option until the full support for remote mode is added.

Reply:
Im working on adding it on the server side, perhaps I should have done it first.

Reply:
I mean if we are tight on schedule we can just checkout the code and keep the functionality the same as previous releases.

Reply:
I think we might be able to make it hopefully within the hour/before EOD